src/main/frontend/components/EditVenue.js

```diff
@@ -0,0 +1,232 @@
+import React, { useState, useEffect } from "react";
+import { Redirect } from "react-router";
+import { useLocation } from "react-router-dom"
+import _ from "lodash";
+
+import StateDropdown from "./StateDropdown";
+import ErrorList from "./ErrorList";
+
+const EditVenue = (props) => {
+  let location = useLocation();
+  const [formPayload, setFormPayload] = useState({
+    name: "",
+    address: "",
+    city: "",
+    state: "",
+    zipCode: "",
+    capacity: "",
+    description: "",
+    imgUrl: "",
+    phoneNumber: ""
+  });
+
+  const [errors, setErrors] = useState({});
+  const [shouldRedirect, setShouldRedirect] = useState(false);
+  const [venueId, setVenueId] = useState(null);
+
+  const fetchVenue = async () => {
+    try {
+      const id = props.match.params.id;
+      setVenueId(id);
+      const response = await fetch(`/api/v1/venues/${id}`);
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`;
+        const error = new Error(errorMessage);
+        throw error;
+      }
+      const responseBody = await response.json();
+      setFormPayload(responseBody.venue);
+    } catch (error) {
+      console.error(`There was an error in fetch: ${error}`);
+    }
+  };
+
+  const editVenue = async () => {
+    try {
+      const response = await fetch(`/api/v1/admin/${venueId}`, {
```

Comment:
I might update the path to something like `api/v1/admin/venues/${venueId}`.  In the future if there are other resources an admin can edit we will want them namespaced appropriately.

Reply:
Agreed. I updated this as suggested